cli/xml2rfc/writers/base.py

```diff
@@ -2250,7 +2250,8 @@ def level_of_section_num(num):
     def is_top_level_section(cls, num):
         return cls.level_of_section_num(num) == 1
 
-    @staticmethod
-    def is_appendix(pn):
+    appendix_pn_re = re.compile(r'^section-[a-z]\.|^section-appendix\.')
```

Comment:
I know this is absurd, but what happens if a document has twenty-seven appendices?   What comes after `section-z`?

Reply:
The system throws an error if there  are too  many:
https://github.com/painless-security/xml2rfc/blob/6041cc1d4b5596d96cf4618826ee5df1c6075c43/cli/xml2rfc/writers/preptool.py#L1347-L1350